### DISCUSSION TITLE
B100D V4 High Rounds

### DISCUSSION THREAD

**Discussion by Vinxent9** [nzp-team/nzportable#541] [Records and Speedruns]
![hr](https://github.com/nzp-team/nzportable/assets/56005655/38031204-fbe9-44cf-8647-12bc03d574cc)

does this still counts? screenshoted after scene transition -_-

**Comment by Kingjaxgaming420** [comment]
random error i got on blood 3 
![image](https://github.com/user-attachments/assets/598424ec-a06f-47f8-be8a-a58a22fb27e6)
